src/classes/UTIL_GlobalProxy.cls

```diff
@@ -0,0 +1,227 @@
+        ERR_Notifier.sendErrorNotifications(context);
+    }
+    
+    /*******************************************************************************************************
+    * @description Prints a debug message to the debug log, if debugging is enabled or a test is being run.
+    * @param message The message to print.
+    * @return void
+    */
+    global static void debug(string message) {
+    	UTIL_Debug.debug(message);
+    }
+    
+   /*******************************************************************************************************
+    * @description Prints a debug message to the debug log, if debugging is enabled or a test is being run, at the specified
+    *              logging level.
+    * @param message The message to print.
+    * @param level The level to output the message at.
+    * @return void
+    */
+    global static void debug(System.LoggingLevel level, string message) {
+    	UTIL_Debug.debug(level, message);
+    }
+    
+    /*******************************************************************************************************
+    * @description Prints a debug message to the debug log, if debugging enabled or a test is being run, with additional info, 
+                   such as heap, cpu, query, and rows usage.
+    * @param message The message to print.
+    * @return void
+    */
+    global static void debugWithInfo(string message) {
+     	UTIL_Debug.debugWithInfo(message);
+    }
+    
+    /*******************************************************************************************************
+    * @description Prints a debug message to the debug log, if debugging is enabled or a test is being run, at the specified logging
+    *              level, with additional info, such as heap, cpu, query, and rows usage.
+    * @param message The message to print.
+    * @param level The level to output the message at.
+    * @return void
+    */
+    global static void debugWithInfo(System.LoggingLevel level, string message) {
+    	UTIL_Debug.debugWithInfo(level, message);
+    }
+    
+    /*******************************************************************************************************
+    * @description Returns object describe data.
+    * @param objectName The name of the object to lookup. 
+    * @return Schema.DescribeSObjectResult Object that contains methods for describing the SObject.
+    */
+    global static Schema.DescribeSObjectResult getObjectDescribe(String objectName) {
+    	return UTIL_Describe.getObjectDescribe(objectName);
+    }
+    
+    /*******************************************************************************************************
+    * @description Returns all field describe data.
+    * @param objectName The name of the object to look up.
+    * @return Map<String, Schema.DescribeFieldResult> A map of object names to describe file results.
+    */  
+    global static Map<String, Schema.DescribeFieldResult> getAllFieldsDescribe(String objectName) {
```

Comment:
@njjc opened #714. I think it's reasonable to expose some useful methods to other packages, etc. Though we should be careful with what to expose. Better too little than too much. 

Regarding why here, and not in its original place, see response above. :-)